### DISCUSSION TITLE
Unable to create airflow user

### DISCUSSION THREAD

**Discussion by sinjup** [OpenLineage/OpenLineage#1040] [Q&A]
Currently I am using airflow version 2.2.5 and wish to upgrade to 2.3.x , while using versions 2.3+ i am unable to create a user. It's showing "No user created  use fab command to create" I have added user creation commands in my entrypoint.sh file. 
![image](https://user-images.githubusercontent.com/110447163/186590326-b9e056e0-0265-40b6-a85f-44a81ac918eb.png)

**Comment by collado-mike** [comment]
Hi, did you mean to open this in the Airflow repo? Or is this somehow OpenLineage related?
